update.sh

```diff
@@ -100,7 +102,21 @@ git remote remove netbsd \
     || die "Failed to remove 'netbsd-src' remote repo"
 
 # Patch bpfjit files to compile.
-git apply patches/bpf.h.patch patches/bpfjit.c.patch patches/bpfjit.h.patch
+git apply --reject \
+    patches/bpf.h.patch \
+    patches/bpfjit.c.patch \
+    patches/bpfjit.h.patch \
+    || die "
+Failed to apply a patch. Correct the patch using following steps:
+1. Apply the *.reject file manually.
+2. Update the failing patch file, for example:
+   git diff src/net/bpf.h > patches/bpf.h.patch
+3. Commit the updated patch file.
+4. Remove the modified netbsd clone.
+   rm -rf _update/netbsd-src
+5. Rerun, but with updated patch files.
+   ./update.sh
```

Comment:
Interesting. So what happens then? The netbsd repo is restored from the backup file, history is rewritten and filtered using git-filter-repo again, git fetch and merge netbsd, then apply patch files again? Maybe we can make a simpler way to do it, like an option to the script which takes us directly to apply patches and commit them...?

Reply:
I'll guess wont run this script that often, but if there is an easier way I'm open for code suggestions.

Reply:
Nah, it's fine.

I'd probably look at the script, then patch and commit. We'll probably do that anyway.